litgpt/pretrain.py

```diff
@@ -369,6 +370,14 @@ def get_dataloaders(
     return train_dataloader, val_dataloader
 
 
+def compute_warmup_iters(devices, max_iters, train, train_dataloader) -> int:
```

Comment:
Can we make this a method in `TrainArgs`? Similarly to how we have `train.gradient_accumulation_iters(devices)`